### ISSUE TITLE
🏁 I'm Lost ↔ v20.0.2

### ISSUE THREAD

**Issue by CREEATION** [itchio/itch-compatibility-watchlist#352]
:rotating_light: game [I'm Lost](https://blackcatstudios.itch.io/im-lost) is broken for me.

  :book: Here's the complete [debug log](https://gist.github.com/f0be103c3d2b7106ba0dfcf1321b9905).

  :running: :(

**Comment by fasterthanlime** [comment]
Looks like nw.js is crashing — is there a file at `C:\Users\CREEATION\AppData\Roaming\itch\apps\Im Lost\win64\icudtl.dat` ? What's its size?

**Comment by CREEATION** [comment]
Nope, there isn't! But in `\win32\` is one, it's around ~10 MB.

**Comment by fasterthanlime** [comment]
wait so there's a `\win32\` folder but no `\win64\` one? the plot thickens!

edit: oh wait you mean there's a `icudtl.dat` in the `win32` folder, but not one in the `win64` one. That sounds like a game bug! Could you post it in the community of the game maybe? https://blackcatstudios.itch.io/im-lost

**Comment by CREEATION** [comment]
Yep, both folders exist but only the `\win32\` one actually has the file you mentioned. Sure, will do!
